packages/http-client/src/remote-admin-api.ts

```diff
@@ -7,40 +7,57 @@ import { StreamID } from '@ceramicnetwork/streamid'
 export class RemoteAdminApi implements AdminApi {
   // Stored as a member to make it easier to inject a mock in unit tests
   private readonly _fetchJson: typeof fetchJson = fetchJson
+
   readonly baseURL = './admin/models'
 
   constructor(
-    private readonly _apiUrl: URL
+    private readonly _apiUrl: URL,
+    private readonly _context: Context
   ) {}
 
   private getUrl(): URL {
     return new URL(this.baseURL, this._apiUrl)
   }
 
+  private async buildAuthorizationHeader(modelsIDs?: Array<StreamID>): Promise<string> {
+    const body = modelsIDs ? modelsIDs.map(streamID => streamID.toString()) : undefined
+    const jws = await this._context.did.createJWS({
+      timestamp: Date.now(),
+      requestPath: this._apiUrl.pathname,
+      requestBody: body
+    })
+    return `${jws.signatures[0].protected}.${jws.payload}.${jws.signatures[0].signature}`
+  }
+
   async addModelsToIndex(modelsIDs: Array<StreamID>): Promise<void> {
     await this._fetchJson(this.getUrl(), {
+      headers: { 'Authorization:': `Basic ${await this.buildAuthorizationHeader(modelsIDs)}` },
```

Comment:
what does the word "basic" in this string do?

Reply:
It's just following the standard for auth headers: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization

I thought that `Basic` is the best choice here.